pallets/liquidity-mining/src/lib.rs

```diff
@@ -0,0 +1,1861 @@
+// This file is part of HydraDX
+
+// Copyright (C) 2020-2021  Intergalactic, Limited (GIB).
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Abbr:
+//  rpvs - reward per valued share
+//  rpz - reward per share in global pool
+
+// Notion spec naming map:
```

Comment:
Can we upgrade the spec so that we remove this? If there are equations that require it, add it to Spec. @dexterslabor @martinfridrich  (not blocking)